src/Native/Runtime/RuntimeInstance.cpp

```diff
@@ -1741,4 +1744,11 @@ COOP_PINVOKE_HELPER(void *, RhAllocateThunksFromTemplate, (PTR_UInt8 moduleBase,
     return pThunkMap;
 }
 
+#ifdef CORERT
```

Comment:
I agree with Jan -- the smaller the hack, the better.  Otherwise, things look good to me.